### PR TITLE
Move RideNaming, RideHeights and max mass to RTDs 

### PR DESCRIPTION
**Pull request by Gymnasiast**

### CODE REVIEW

File: src/openrct2/ride/coaster/meta/FlyingRollerCoaster.h [hunk 1]
```diff
@@ -55,6 +56,7 @@ constexpr const RideTypeDescriptor FlyingRollerCoasterAltRTD =
                      RIDE_TYPE_FLAG_START_CONSTRUCTION_INVERTED),
     SET_FIELD(RideModes, (1ULL << RIDE_MODE_CONTINUOUS_CIRCUIT) | (1ULL << RIDE_MODE_CONTINUOUS_CIRCUIT_BLOCK_SECTIONED)),
     SET_FIELD(DefaultMode, RIDE_MODE_CONTINUOUS_CIRCUIT),
+    SET_FIELD(Naming, { STR_RIDE_NAME_3A, STR_RIDE_DESCRIPTION_UNKNOWN }),
```
Comment (tupaschoal) [inline src/openrct2/ride/coaster/meta/FlyingRollerCoaster.h:59]: Maybe a TODO to create those (or just reuse from the non alt?)

File: src/openrct2/ride/coaster/meta/LayDownRollerCoaster.h [hunk 2]
```diff
@@ -51,6 +52,7 @@ constexpr const RideTypeDescriptor LayDownRollerCoasterAltRTD =
                      RIDE_TYPE_FLAGS_COMMON_COASTER | RIDE_TYPE_FLAG_HAS_LARGE_CURVES),
     SET_FIELD(RideModes, (1ULL << RIDE_MODE_CONTINUOUS_CIRCUIT) | (1ULL << RIDE_MODE_CONTINUOUS_CIRCUIT_BLOCK_SECTIONED)),
     SET_FIELD(DefaultMode, RIDE_MODE_CONTINUOUS_CIRCUIT),
+    SET_FIELD(Naming, { STR_RIDE_NAME_40, STR_RIDE_DESCRIPTION_UNKNOWN }),
```
Comment (tupaschoal) [inline src/openrct2/ride/coaster/meta/LayDownRollerCoaster.h:55]: Same

File: src/openrct2/ride/coaster/meta/MultiDimensionRollerCoaster.h [hunk 3]
```diff
@@ -52,6 +53,7 @@ constexpr const RideTypeDescriptor MultiDimensionRollerCoasterAltRTD =
                      RIDE_TYPE_FLAGS_COMMON_COASTER | RIDE_TYPE_FLAG_HAS_LARGE_CURVES),
     SET_FIELD(RideModes, (1ULL << RIDE_MODE_CONTINUOUS_CIRCUIT) | (1ULL << RIDE_MODE_CONTINUOUS_CIRCUIT_BLOCK_SECTIONED)),
     SET_FIELD(DefaultMode, RIDE_MODE_CONTINUOUS_CIRCUIT),
+    SET_FIELD(Naming, { STR_RIDE_NAME_38, STR_RIDE_DESCRIPTION_UNKNOWN }),
```
Comment (tupaschoal) [inline src/openrct2/ride/coaster/meta/MultiDimensionRollerCoaster.h:56]: Same

File: src/openrct2-ui/windows/Ride.cpp [hunk 4]
```diff
@@ -2262,6 +2262,33 @@ static void window_ride_show_open_dropdown(rct_window* w, rct_widget* widget)
...
+        case RIDE_TYPE_52:
+            return STR_RIDE_NAME_52;
+        case RIDE_TYPE_53:
+            return STR_RIDE_NAME_53;
+        case RIDE_TYPE_54:
+            return STR_RIDE_NAME_54;
+        case RIDE_TYPE_55:
+            return STR_RIDE_NAME_55;
+        case RIDE_TYPE_59:
+            return STR_RIDE_NAME_59;
```
[the remark this answers is not in the document]
Comment (Gymnasiast) [reply src/openrct2-ui/windows/Ride.cpp:2286]: Only in the ride type dropdown, really, which is why I created this bit. It should go away once we start doing something with these unused ride types.

**Pull request merged**